featuretools/tests/utils_tests/test_utils_info.py

```diff
@@ -14,6 +18,17 @@ def this_dir():
     return os.path.dirname(os.path.abspath(__file__))
 
 
+def test_show_info():
+    f = io.StringIO()
+    with redirect_stdout(f):
```

Comment:
pytest has a [capsys fixture](https://docs.pytest.org/en/latest/how-to/capture-stdout-stderr.html#accessing-captured-output-from-a-test-function) to capture output in a test, I think it'd be good to use the pytest approach

Reply:
Woah, did not know about this. Fixed
https://github.com/alteryx/featuretools/pull/2353/commits/3cba6ffb7854dcadbc14a1533042d4878fe2f1da